package/CHANGELOG

```diff
@@ -13,7 +13,7 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
-??/??/16 jandom, abhinavgupta94, orbeckst
+??/??/16 jandom, abhinavgupta94, orbeckst, kain88-de
```

Comment:
Did you use some of @hainm 's work from PR #571 ? If so, also add him to the list.